src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -540,6 +544,9 @@ def _populate_sql_container_definition(sql_container_resource,
     if unique_key_policy is not None:
         sql_container_resource.unique_key_policy = unique_key_policy
 
+    if client_encryption_policy is not None:
+        _validate_and_populate_client_encryption_policy(sql_container_resource, client_encryption_policy)
```

Comment:
You can add the validator in parameter definition then just simply pass the value in custom.py